swapi/src/scripts/dataModel.js

```diff
@@ -0,0 +1,39 @@
+class Model {
+    constructor() {
+		this.resultVolume = 5;
```

Comment:
Could you align your code please? Remove an extra spaces on the left side everywhere

Reply:
I deleted extra spaces there and in other places.
```suggestion
class Model {
  constructor() {
    this.resultVolume = 5;
  }
```